packages/core-modules/contracts/test/modules/UpgradeModuleMock.sol

```diff
@@ -4,13 +4,7 @@ pragma solidity ^0.8.0;
 import "../../modules/UpgradeModule.sol";
 
 contract UpgradeModuleMock is UpgradeModule {
```

Comment:
In an integrated system, we would set the first owner via the OwnerModule.

Since we're testing the UpgradeModule in isolation here, we need to mock a way to set the first owner.